cmd/vcert/validators.go

```diff
@@ -745,6 +745,15 @@ func validateProvisionFlags(commandName string) error {
 		}
 	}
 
+	if flags.certificateIDFile != "" {
+		if flags.pickupID != "" {
```

Comment:
is this IF out of place? it is checking pickup ID, not certificate ID...

Reply:
Indeed it is checking the pickup-id, in case user getsfunny and tries to specify the pickupID and the certificate ID File at the same time, hence the error message:
`both --certificate-id and --pickup-id-file options cannot be specified at the same time`

Below is the check for the certificate ID itself